docs/rules/no-export.md

```diff
@@ -0,0 +1,51 @@
+# no export from test file (no-export)
+
+I ran into this issue when we had a set of helper functions in 1 test file that
```

Comment:
While it's really useful to understand why you want this rule, I don't think we need it in the docs

Reply:
fair enough.  fixed.